packages/react/src/MegaMenu/MegaMenu.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { MegaMenu } from './MegaMenu'
+import '@testing-library/jest-dom'
+
+describe('Mega menu', () => {
+  it('renders', () => {
+    const { container } = render(<MegaMenu />)
+
+    const component = container.querySelector(':only-child')
```

Comment:
Try to use the ARIA roles for component selection, so `getByRole('navigation')` in this case (if we choose to keep this a `nav`, see other comment)

Reply:
Sure, I did have to revert to using a `nav` element, which I would prefer.